tests/test_evolve.py

```diff
@@ -16,4 +16,3 @@ def test_sv_scheme():
     assert pineko.evolve.sv_scheme(schemeA_tcard) == "exponentiated"
     assert pineko.evolve.sv_scheme(schemeB_tcard) == "expanded"
     assert pineko.evolve.sv_scheme(schemeC_tcard) == None
```

Comment:
```suggestion
    assert pineko.evolve.sv_scheme(schemeC_tcard) is None
```

I know it's not this PR fault, but since you touched the file I saw it :P